src/activities/components/ActivityUsers.story.js

```diff
@@ -45,10 +55,11 @@ storiesOf('ActivityUsers', module)
       activity: {
         ...leavableActivity,
         participants: [
-          leavableActivity.participants.find(c => c.isCurrentUser),
-          ...leavableActivity.participants.filter(c => !c.isCurrentUser),
+          leavableActivity.participants.find(c => c.user.isCurrentUser),
+          ...leavableActivity.participants.filter(c => !c.user.isCurrentUser),
```

Comment:
`c` is participant? could also destructure `user` to avoid these single-purpose variables ;)

Reply:
this whole story is broken, users don't have `isCurrentUser` anyway... I didn't migrate/check the *.story.js files... will have a look, although maybe not too long :)

Reply:
I'll leave this story out for now :)